deployer/conf/elasticsearch.yml

```diff
@@ -58,6 +58,7 @@ path:
 searchguard:
   authcz.admin_dn:
   - CN=system.admin,OU=OpenShift,O=Logging
+  cert.oid: '1.3.6.1.4.1.2312.17.100.2.1'
```

Comment:
This change will only replace the default value we were using with the one provided, which is also baked into the self generated certs that we do now.  This only moves us away from using the default OID using the mechanism that is the default for SG.  We will need to produce an alternative impl to https://github.com/floragunncom/search-guard/pull/269/files#diff-69afaa0c3b3e2095d1cefe863b648fb7R42 in order for us to validate any extension value properly.  Additionally we will need to consume the cert service which we have yet to do.